### PR TITLE
image carousel

### PR DESCRIPTION
**Pull request by ckcomin**

### CODE REVIEW

File: src/index.html
Comment (spencer-rafada) [inline src/index.html:1]: Looks great!

File: src/mjs/main.js
Comment (spencer-rafada) [inline src/mjs/main.js:1]: Don't forget to run **npm run format** before committing next time. Great job on the carousel!

File: src/styles/global.css
Comment (spencer-rafada) [inline src/styles/global.css:1]: Can you merge our main branch right now so the styles won't be a problem with vite? If you need help with this, contact me and I can help you!

The steps will usually be:
1. Go to your branch
2. On your console, write git merge main
3. Then you will resolve all of the conflicts from main to your branch.

**Review by spencer-rafada — CHANGES_REQUESTED** [review]
The carousel looks great! I have not been able to explain this before, but you can merge new versions of main to your branch when we make changes to main while you are working on your branch and have not finished it. I'll gladly walk you through this process one of these days if you are free!

File: src/styles/global.css
Comment (ckcomin) [reply src/styles/global.css:1]: Okay I just did it

File: src/mjs/main.js
Comment (ckcomin) [reply src/mjs/main.js:1]: I ran the npm run format and i got "[error] No files matching the pattern were found: "'./**/*.{js,jsx,ts,tsx,css,md,json}'"."
This error also shows on the main.

Comment (spencer-rafada) [reply src/mjs/main.js:1]: That just means there's no file types like that in the repository. That's fine.

File: src/styles/global.css
Comment (spencer-rafada) [reply src/styles/global.css:1]: Thank you! Lastly, make sure all the JavaScript file in the mjs folder are placed inside the js folder and linked correctly in the HTML. Besides that, everything else looks good.

**Review by spencer-rafada — APPROVED** [review]

**Pull request merged**